timeplus/client.go

```diff
@@ -359,39 +362,38 @@ func (s *TimeplusClient) QueryV2Stream(sql string, count int, bufferTime int) (r
 				colonIndex := strings.Index(dataLine, ":")
 				eventContentData := dataLine[colonIndex+1:]
 				if eventData == "query" {
-					var m QueryEvent
+					var m queryEvent
 					err := json.Unmarshal([]byte(eventContentData), &m)
 					if err != nil {
 						ch <- rxgo.Error(fmt.Errorf("invalide sse response,%s, %s", err, eventContentData))
 					}
 
-					event := &ServerSideEvent{
-						EventType: "query",
-						QueryData: &m,
+					event := &serverSentEvent{
+						eventType: "query",
+						queryData: &m,
 					}
 					ch <- rxgo.Of(event)
 				} else if eventData == "metrics" {
-					var m MetricsEvent
+					var m metricsEvent
 					err := json.Unmarshal([]byte(eventContentData), &m)
 					if err != nil {
 						ch <- rxgo.Error(fmt.Errorf("invalide sse response,%s, %s", err, eventContentData))
 					}
-					event := &ServerSideEvent{
-						EventType:   "metrics",
-						MetricsData: &m,
+					event := &serverSentEvent{
+						eventType:   "metrics",
+						metricsData: &m,
 					}
 					ch <- rxgo.Of(event)
 				}
-			} else if eventField == "data" {
-
+			} else {
 				var m DataEvent
 				err := json.Unmarshal([]byte(eventData), &m)
 				if err != nil {
 					ch <- rxgo.Error(fmt.Errorf("invalide sse response, %s", line))
 				}
-				event := &ServerSideEvent{
-					EventType: "data",
-					Data:      &m,
+				event := &serverSentEvent{
+					eventType: "data",
```

Comment:
This is confusing. In our implementation, there is no event type for the query result data. Having `eventType: "data"` here may mislead the user to think the payload is
```
event: data
data: [[],[]]

event: data
data: [[],[]]
```

However, our actual payload is just
```
data: [[],[]]

data: [[],[]]

data: [[],[]]
```

I'd suggest to just leave `eventType: ""` here